lib/silfont/gfr.py

```diff
@@ -239,7 +239,7 @@ def write(self, filename=None): # Write data to a file
 def setpaths(logger): # Check that the script is being run from the root of the repository and set standard paths
     repopath = os.path.abspath(os.path.curdir)
     # Do cursory checks that this is the root of the fonts repo
-    if repopath[-6:] != "/fonts" or not os.path.isdir(os.path.join(repopath, "fonts/sil")):
+    if repopath[-5:] != "fonts" or not os.path.isdir(os.path.join(repopath, "fonts/sil")):
```

Comment:
Since this is a cursory check, it seems checking for "fonts" should be enough, rather than for "/fonts" or "\fonts".